user-ops-indexer/user-ops-indexer-logic/src/indexer/v06/indexer.rs

```diff
@@ -0,0 +1,506 @@
+
+        client.push(receipt).unwrap();
+        client.push(tx).unwrap();
+
+        let indexer = IndexerV06::new(Provider::new(PubSubMockProvider(client)), db);
+        indexer.handle_tx(tx_hash).await.unwrap();
```

Comment:
Since handle_tx should update database I guess it's better to check that database contains right rows